app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,35 @@
+class NotesController < ApplicationController
+
+def index
+  @notes = Note.all
+  render json: @notes
+end
+
+def create
+  @note= Note.new(note_params)
+  if @note.save
+    check_tags
+    render json: @note
+  else
+    render json: @note.errors
```

Comment:
Need to set `status: 400` here.

And the test is expecting something in the form of:

```
{ errors: 
  [
    {error: "Title cannot be blank"},
    {error: "Body cannot be blank"}
  ]
}
```